### DISCUSSION TITLE
Setting a npmRegistryServer for a private package without a scope?

### DISCUSSION THREAD

**Discussion by ericrav** [yarnpkg/berry#2702] [Q&A] [answered]
Is it or should it be possible to configure a different `npmRegistryServer` for a specific package? We have a private package that doesn't include a `@scope`, so I don't think `npmScopes` in `.yarnrc.yml` works.

**Comment by arcanis** [comment, marked as answer]
> Is it possible to configure a different `npmRegistryServer` for a specific package?

Kinda. You can use the `resolutions` field to force the download from a specific location. You can also write a resolver plugin that handles this one particular package.

> We have a private package that doesn't include a @scope, so I don't think npmScopes in .yarnrc.yml works.

We don't really support this use case natively, and I'm not convinced it's common enough (and difficult enough to workaround) that it's worth the extra complexity 🤔

**Reply by ericrav** [reply]
Thanks for the info. I thought `resolutions` could only force a version—can you show what you mean by `force the download from a specific location`. Or do you mean that would only be possible by setting it to a custom protocol/resolver?

**Reply by arcanis** [reply]
> I thought `resolutions` could only force a version

You can force a url too, meaning that you could make it point directly to the tarball url.

**Reply by ericrav** [reply]
Oh gotcha—thanks!
